crypto/hash/auth.c

```diff
@@ -138,10 +136,11 @@ srtp_auth_type_test (const srtp_auth_type_t *at, const srtp_auth_test_case_t *te
                     srtp_octet_string_hex_string(test_case->key,
                                                  test_case->key_length_octets));
         debug_print(srtp_mod_auth, "data: %s",
-                    srtp_octet_string_hex_string(test_case->data,
-                                                 test_case->data_length_octets));
-        debug_print(srtp_mod_auth, "tag computed: %s",
-                    srtp_octet_string_hex_string(tag, test_case->tag_length_octets));
+                    srtp_octet_string_hex_string(
+                        test_case->data, test_case->data_length_octets));
+        debug_print(
+            srtp_mod_auth, "tag computed: %s",
+            srtp_octet_string_hex_string(tag, test_case->tag_length_octets));
```

Comment:
That similarish things end up looking different - no way around that really.
I might have an idea of making them slightly more uniform - breaking the line for each and every argument to a function - but need to read up on my clang-format foo in order to do that.